src/migrations/20200903150725-create-culture.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+module.exports = {
```

Comment:
since w are uisng uuid, this block of code should be 
```
  
export default {
  up(queryInterface, Sequelize) {
    return queryInterface.sequelize.query('CREATE EXTENSION IF NOT EXISTS "uuid-ossp";').then(() => queryInterface.createTable('Users', {
```

That is just an example....

Reply:
done